pilot/pkg/xds/ads.go

```diff
@@ -147,11 +147,19 @@ func (s *DiscoveryServer) receive(con *Connection, identities []string) {
 		req, err := con.stream.Recv()
 		if err != nil {
 			if istiogrpc.IsExpectedGRPCError(err) {
-				log.Infof("ADS: %q %s terminated", con.peerAddr, con.conID)
+				if req != nil && req.TypeUrl == v3.SecretType {
+					log.Infof("ADS: connection terminated for secrets:%v", strings.Join(req.ResourceNames, ","))
```

Comment:
Secret is used even for Istiod, and may have 10000s of secrets, not sure we should log this